jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -343,6 +345,11 @@ def collapse_to_spectrum(self, add_data=True, **kwargs):
                 sender=self)
             self.hub.broadcast(snackbar_message)
 
+        # per https://jwst-docs.stsci.edu/jwst-near-infrared-camera/nircam-performance/nircam-absolute-flux-calibration-and-zeropoints # noqa
+        if 'PIXAR_SR' in spectral_cube.meta:
+            pix_scale_factor = self.aperture.scale_factor * 10e6 * spectral_cube.meta['PIXAR_SR']
```

Comment:
where does the 10^6 come from?  Does that make assumptions on units (converting between MJy and Jy maybe)?

Reply:
I had made the assumption based on conversations from the hack day and the meetings with Patrick that yes MJy to Jy would need scientific notation conversion, I'll remove it from here and the specification in the `translate_units()` function

Reply:
it would, but we're going to want to have the flexibility to change between any prefix.  Ideally this would take the new units, but if that's too difficult, we can do that as a followup and just assume the same units (except the sr) otherwise.